report/string_set.go

```diff
@@ -54,33 +54,35 @@ func (s StringSet) Intersection(b StringSet) StringSet {
 // Add adds the strings to the StringSet. Add is the only valid way to grow a
 // StringSet. Add returns the StringSet to enable chaining.
 func (s StringSet) Add(strs ...string) StringSet {
+	r := s.Copy()
```

Comment:
I think it would be completely acceptable to do:

```
r := make([]string, len(s)+len(strs))
copy(r, s)
```

Since slices grow exponentially on append() anyways. But, precisely for that reason it may not bring too far.